data-manipulation/hashing/md5/hash-data-with-md5.yml

```
@@ -30,3 +31,6 @@ rule:
         - and:
           - number: 0x8003 = CALG_MD5
           - api: advapi32.CryptCreateHash
+      - and:
+        - api: System.Security.Cryptography.MD5::Create
+        - api: System.Security.Cryptography.HashAlgorithm::ComputeHash
```

```suggestion
      - and:
        - format: dotnet
        - api: System.Security.Cryptography.MD5::Create
        - api: System.Security.Cryptography.HashAlgorithm::ComputeHash
```

not required, but helps to document where these APIs come from.